go.mod

```diff
@@ -106,4 +106,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-replace github.com/equinix-labs/fabric-go v0.4.0 => github.com/manu-equinix/fabric-go v0.4.1-alpha.0.20230501231222-1857e108d65a
+//replace github.com/equinix-labs/fabric-go v0.4.0 => github.com/manu-equinix/fabric-go v0.4.1-alpha.0.20230501231222-1857e108d65a
+replace github.com/equinix-labs/fabric-go v0.4.0 => ../fabric-go
```

Comment:
Why do we have a replace command for equinix-labs/fabric-go here?

Reply:
we'll keep this change in our local setups